release-notes/7.0/known-issues.md

```diff
@@ -54,6 +54,19 @@ More information and workaround can be found at https://github.com/dotnet/runtim
 
 ## .NET SDK
 
+### [Testhost.exe does not find '7.0.0-preview.6.22324.4' runtime when running tests](https://github.com/dotnet/sdk/issues/26462)
+A file-based install of the SDK, dotnet test on a .net 7 project will look in the global location rather than the local location and be unable to find .net 7.
+
+Workaround: Setting DOTNET_ROOT to point to 'C:\Users\adity\AppData\Local\Microsoft\dotnet' fixes the issue.
+
+### Blazor issues depending on whether you’re using .NET 6 installed by VS or stand-alone install of .NET 7
+
+If your app is targetting Blazor using the .net 7 included in VS, you can target net6.0 but not target net7.0. 
```

Comment:
Add _You will see an error message 'The "ProcessFrameworkReferences" task faield unexpectedly.'_